.github/workflows/continuous-integration-workflow.yml

```diff
@@ -75,11 +77,11 @@ jobs:
                 -DKokkos_ENABLE_OPENMP=ON && \
               cmake --build builddir --parallel 2 && \
               ctest --test-dir builddir --output-on-failure"
+          sed -i '26s/$/ /' ${{ matrix.config.dockerfile }} # workaround for kokkos/ci-containers#56 
```

Comment:
What is that doing and why is it needed?

Reply:
As a workaround we are adding a space to the end of the line to force docker to "rebuild" line 26 and later commands.